extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AuthConfig.java

```diff
@@ -31,4 +31,13 @@ public interface AuthConfig {
      */
     @WithDefault("true")
     boolean proactive();
+
+    /**
+     * Propagate security identity to support the `SecurityIdentity` CDI bean injection.
+     * Enable this option if you try to inject the identity inside a Vert.x route handler
```

Comment:
```suggestion
     * Enable this option if you inject the identity from a Vert.x route handler or a Reactive Route.
```

(and remove the rest of the sentence)

Reply:
done